solr/core/src/java/org/apache/solr/packagemanager/SolrPackage.java

```diff
@@ -0,0 +1,107 @@
+package org.apache.solr.packagemanager;
+
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
```

Comment:
use `org.apache.solr.common.annotation.JsonProperty`

Reply:
I wasn't able to. None of the serialization/deserialization stuff worked.